.github/stale.yaml

```diff
@@ -0,0 +1,19 @@
+name: "Stale issue handler"
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
```

Comment:
Add a comment with info and a link to the documentation for this GitHub action works.